pyomo/solvers/plugins/solvers/cplex_direct.py

```diff
@@ -544,6 +544,14 @@ def _postsolve(self):
             self.results.solver.status = SolverStatus.aborted
             self.results.solver.termination_condition = TerminationCondition.maxIterations
             soln.status = SolutionStatus.stoppedByLimit
+        elif status in {
+            rtn_codes.solution_limit,
+            rtn_codes.node_limit_feasible,
+            rtn_codes.mem_limit_feasible
+        }:
+            self.results.solver.status = SolverStatus.aborted
+            self.results.solver.termination_condition = TerminationCondition.unknown
```

Comment:
I think that would be a design decision for the Pyomo maintainers. For the time being I'd like to just reconcile the two solvers rather than introduce new termination conditions (which no other solver would then be using).